ui-tests/pom.xml

```
@@ -33,14 +33,29 @@
         <artifactId>jenkins-core</artifactId>
         <version>${jenkins.version}</version>
       </dependency>
+	<!-- need to match any BOM imports from ATH due to https://issues.apache.org/jira/browse/MNG-5761 -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-bom</artifactId>
+        <version>4.1.91.Final</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>2.14.2</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 
   <dependencies>
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>acceptance-test-harness</artifactId>
-      <version>5504.v485694f31cdf</version>
+      <version>5535.v48093a_7cc4a_c</version>
```

updated version https://github.com/jenkinsci/acceptance-test-harness/pull/1101
```suggestion
      <version>5536.v62b_b_4b_a_49e65/</version>
```